WORKSPACE

```
@@ -189,3 +189,10 @@ maven_install(
     strict_visibility = True,
     version_conflict_policy = "pinned"
 )
+
+# Load JetBrains Compose for Kotlin.
+load("@vaticle_dependencies//builder/compose:deps.bzl", compose_deps = "deps")
+
+compose_deps()
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_binary", "kt_jvm_library")
```

Missed a couple of newlines
```suggestion
load("@vaticle_dependencies//builder/compose:deps.bzl", compose_deps = "deps")
compose_deps()
load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_binary", "kt_jvm_library")
```